backend/tests/tests.js

```diff
@@ -0,0 +1,12 @@
+import config from '../config/config';
```

Comment:
Nitpick: The config file could be more concise and not the same name as directory

Reply:
I named the config file index.js as this allows all imports to be simplified to just /config